Fetch/Fetch.php

```diff
@@ -209,6 +317,30 @@ public function search()
         return $this->handle($data);
     }
 
+    /**
+     * Submit a form
+     */
+    public function form($name)
```

Comment:
Just a few things I noticed while taking a short look on the PR

There are quite a few possible improvements to this endpoint
- support for [honeypot, email sending, file uploads](https://docs.statamic.com/forms)
- support for [events](https://docs.statamic.com/addons/events#events), required to work with plugins like [recaptcha](https://github.com/aryehraber/statamic-recaptcha)